docs/v4/start/installation.md

```diff
@@ -4,7 +4,6 @@ title: Installation
 
 ## System Requirements
 
-* Web server with URL rewriting
```

Comment:
Mmm this is still a requirement. it might sort of be satisfied by the built-in web server. but the built-in web server should never be used in production.